hooks/command

```diff
@@ -38,25 +41,35 @@ docker \
     --volume "$PLUGIN_DIR/ruby:/src" \
     --env "BUILDKITE_PLUGIN_JUNIT_ANNOTATE_JOB_UUID_FILE_PATTERN=${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_JOB_UUID_FILE_PATTERN:-}" \
     --env "BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAILURE_FORMAT=${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAILURE_FORMAT:-}" \
+    --env "BUILDKITE_PLUGIN_JUNIT_ANNOTATE_REPORT_SLOWEST=${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_REPORT_SLOWEST:-}" \
     ruby:2.7-alpine ruby /src/bin/annotate /junits \
       > "$annotation_path"
 
+if [[ $? -eq 64 ]]; then # special exit code to signal test failures
+  annotation_style="error"
+  if [[ "${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAIL_BUILD_ON_ERROR:-false}" =~ (true|on|1) ]]; then
+    fail_build=1
+  fi
+fi
+
+set -e
+
 cat "$annotation_path"
 
 if grep -q "<details>" "$annotation_path"; then
-
   if ! check_size; then
-    echo "--- :warning: Failures too large to annotate"
-    echo "The failures are too large to create a build annotation. Please inspect the failed JUnit artifacts manually."
+    echo "--- :warning: Report too large to annotate"
```

Comment:
Could we leave the 'Report' as 'failures'?

Same for the changes below.

Reply:
Well I thought that would be confusing since the annotation will usually not consist of failures with that setting enabled. But I'll change it if you don't think that's an issue, of course.

Reply:
I've added a commit that reverses the mentioned changes, just in case.